www/include/options/accessLists/resourcesACL/formResourcesAccess.php

```diff
@@ -40,116 +40,115 @@
 /*
  * Database retrieve information for LCA
  */
-if ($o == "c" || $o == "w") {
+if ($o === 'c' || $o === 'w') {
     /*
      * Set base value
      */
-    $DBRESULT = $pearDB->query("SELECT * FROM acl_resources WHERE acl_res_id = '" . $acl_id . "' LIMIT 1");
-    $acl = array_map("myDecode", $DBRESULT->fetchRow());
+    $statement = $pearDB->prepare("SELECT * FROM acl_resources WHERE acl_res_id = :aclId LIMIT 1");
+    $statement->bindValue(':aclId', $aclId, \PDO::PARAM_INT);
+    $statement->execute();
+    $acl = array_map("myDecode", $statement->fetch());
 
     /*
      * Set Poller relations
      */
-    $query = "SELECT poller_id FROM acl_resources_poller_relations WHERE acl_res_id = '" . $acl_id . "'";
-    $DBRESULT = $pearDB->query($query);
-    for ($i = 0; $pollers_list = $DBRESULT->fetchRow(); $i++) {
-        $acl["acl_pollers"][$i] = $pollers_list["poller_id"];
+    $statement = $pearDB->prepare("SELECT poller_id FROM acl_resources_poller_relations WHERE acl_res_id = :aclId");
+    $statement->bindValue(':aclId', $aclId, \PDO::PARAM_INT);
+    $statement->execute();
+    while ($poller = $statement->fetch()) {
+        $acl["acl_pollers"][] = $poller["poller_id"];
     }
-    $DBRESULT->closeCursor();
 
     /*
      * Set Hosts relations
      */
-    $hostnotexludes = array();
-    $query = "SELECT host_host_id FROM acl_resources_host_relations WHERE acl_res_id = '" . $acl_id . "'";
-    $DBRESULT = $pearDB->query($query);
-    for ($i = 0; $hosts_list = $DBRESULT->fetchRow(); $i++) {
-        $acl["acl_hosts"][$i] = $hosts_list["host_host_id"];
-        $hostnotexludes[$hosts_list["host_host_id"]] = 1;
+    $hostnotexludes = [];
```

Comment:
Maybe use the camelCase here :)

Reply:
Unless it is used everywhere with some includes of this file in some other files inception style

Reply:
removed :) it was not used...